extensions/compose/src/cli-run.ts

```diff
@@ -44,7 +44,7 @@ export function getSystemBinaryPath(binaryName: string): string {
 }
 
 // Takes a binary path (e.g. /tmp/docker-compose) and installs it to the system. Renames it based on binaryName
-export async function installBinaryToSystem(binaryPath: string, binaryName: string): Promise<void> {
+export async function installBinaryToSystem(binaryPath: string, binaryName: string): Promise<string | undefined> {
```

Comment:
should not be Promise<string | undefined> the signature should always return a string, if the function was not able to install system-wide (user cancel, or refuse to give admin privilege) it should throw an error

Reply:
@axel7083 I'll open follow up on this, because I don't want add too much new functionality to the fix.